packages/suite/src/views/wallet/sign-verify/components/SignMessage/index.tsx

```diff
@@ -0,0 +1,249 @@
+        flex: 1 1 100%;
+    }
+`;
+
+const StyledSelect = styled(Select)`
+    padding: 0;
+    margin-bottom: 0px;
+    @media screen and (max-width: ${variables.SCREEN_SIZE.SM}) {
+        font-size: ${variables.FONT_SIZE.TINY};
+    }
+`;
+
+const Row = styled.div`
+    padding-bottom: 28px;
+    display: flex;
+`;
+const RowButtons = styled(Row)`
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+`;
+const StyledButton = styled(Button)`
+    width: 110px;
+    margin-left: 10px;
+
+    &:first-child {
+        margin-left: 0;
+    }
+`;
+const LeftSideButton = styled(StyledButton)`
+    margin-right: auto;
+`;
+
+interface AddressToSign {
+    address: string;
+    path: string;
+    accountsSelectOpt?: Array<{ label: string; value: string }>;
+}
+type errorsMessage = 'TR_ADDRESS_IS_NOT_VALID' | 'TR_REQUIRED_FIELD';
+type Inputs = {
+    address: string;
+    path: string;
+    signMessage: string;
+    accountsSelect: { label: string; value: string; type: '' };
+    signature: string;
+};
+
+let path: AddressToSign['address'] = ''; // fast click on copy button failed if put this inside a component
+let address: AddressToSign['path'] = '';
+let accountsSelectOpt: AddressToSign['accountsSelectOpt'];
+
+const SignMessage = ({ account, isLocked }: Props) => {
+    const { sign, openModal } = useActions({
+        sign: signVerifyActions.sign,
+        openModal: modalActions.openModal,
+    });
+    if (
+        account.networkType === 'bitcoin' &&
+        typeof account.addresses?.unused[0]?.address === 'string' &&
+        typeof account.addresses?.unused[0]?.path === 'string'
+    ) {
+        const accountsSelectArr = [...account.addresses.unused, ...account.addresses.used];
+        accountsSelectOpt = accountsSelectArr.map(el => ({
+            label: el.address,
+            value: el.path,
+        }));
+    } else if (account.networkType === 'ethereum') {
+        address = account.descriptor;
+        path = account.path;
+    }
+
+    const { getValues, setValue, errors, formState, reset, control, register, watch } = useForm<
+        Inputs
+    >({
+        defaultValues: {
+            address,
+            path,
+            signMessage: '',
+            accountsSelect: { label: '', value: '', type: '' },
+            signature: '',
+        },
+        mode: 'onChange',
+    });
+    const { isValid } = formState;
+    const watchSignature = watch('signature');
+
+    useEffect(() => {
+        return () => {
+            reset(); // clear data after coin change
+            path = '';
+            address = '';
+        };
+    }, [reset]);
+
+    return (
+        <Card title={<Translation id="TR_SIGN_MESSAGE" />} isColumn>
+            <Column>
+                <Row>
+                    {account.networkType === 'bitcoin' && (
+                        <Controller
+                            as={<StyledSelect options={accountsSelectOpt} />}
```

Comment:
In bitcoin case (got multiple options) `Select` should have preselected first unused address
There is no possibility to have "empty" address field in select. You can only sign message with your own address/path (and we know them all), there is no other way to do this...

Reply:
I don't think it's obvious, I didn't have a exactly described task, ok I'll fix it

Reply:
i'm pretty sure that i've told you that multiple times...

Reply:
"In bitcoin case (got multiple options) Select should have preselected first unused address" - no one told me that, and it's bad form to give voice tasks in IT and then claim that you said it